lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1798,6 +1800,7 @@ sub cancel {
     my $result = $obsoleted ? OBSOLETED : USER_CANCELLED;
     return if ($self->result ne NONE);
     my $state = $self->state;
+    $self->release_networks;
     $self->update(
```

Comment:
I guess it would be safer (for later) to make it pass for done() as well

Reply:
I was thinking the same, but didn't dare the possible side effects